docs/feature_flags.md

```diff
@@ -125,7 +125,40 @@ histogram (albeit via the text format). With this flag enabled, Prometheus will
 still ingest those conventional histograms that do not come with a
 corresponding native histogram. However, if a native histogram is present,
 Prometheus will ignore the corresponding conventional histogram, with the
-notable exception of exemplars, which are always ingested.
+notable exception of exemplars, which are always ingested. To keep the
+conventional histograms as well, enable `scrape_classic_histograms` in the
+scrape job.
+
+NOTE: Due to the protobuf serialization, conventional histogram `le` labels
+and summary `quantile` labels will be formatted differently. Any integer
```

Comment:
It doesn't change in general. It only changes if the classical text format is used (and not OpenMetrics) _and_ the instrumentation library used by the target formats numbers differently than in OpenMetrics. You explain that at the end of the paragraph, but at that time, many readers have already needlessly panicked. I would propose to open the paragraph with the description of the situation where the rest is even relevant. Something like: "In certain situation, the protobuf parsing changes the number formatting of the `le` labels of conventional histograms and the `quantile` labels of summaries. Typically, this happens if the scraped target is instrumented with [client_golang](https://github.com/prometheus/client_golang) provided that [promhttp.HandlerOpts.EnableOpenMetrics](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus/promhttp#HandlerOpts)
is set to false. …" And then continue with the detailed explanation of the format changes.